SimulationAgent/DeviceConnection/DeviceConnectionActor.cs

```diff
@@ -124,7 +116,11 @@ public enum ActorEvents
         /// <summary>
         /// Failed device twin updates counter
         /// </summary>
-        public long FailedTwinUpdatesCount => this.failedTwinUpdatesCount;
+        public long FailedTwinUpdatesCount
+        {
+            get => this.failedTwinUpdatesCount;
+            set => this.failedTwinUpdatesCount = value;
```

Comment:
@jillcary Could you help me understand why a setter needed for `FailedTwinUpdatesCount` ?

Reply:
Instead of adding a circular dependency on the `DeviceConnectionActor` from the `DevicePropertiesActor` if we add a setter for the `FailedTwinUpdatesCount`, then the `DevicePropertiesActor` can increment the `FailedTwinUpdatesCount` in the connection actor and the connection actor can still maintain the summed error count.

Another option instead of a setter would be a public method like "IncrementFailedTwinUpdateCount".

Reply:
@jillcary see my other comment about `SimulationErrorsCount`, I think we can avoid this setter, updating the SimulationRunner to pull error counters out of each actor